source/iface/platform.rst

```diff
@@ -31,42 +36,58 @@ A platform contains 1 or more SYCL devices, or a host device.
 ::
 
   platform();
-  explicit platform(cl_platform_id platformID);
-  explicit platform(const sycl::device_selector &deviceSelector);
+  template <typename DeviceSelector> explicit platform(const DeviceSelector&);
 
 Construct a SYCL platform instance.
 
-The default constructor creates a host platform. When passed a
-``cl_platform_id``, an OpenCL|trade| platform is used to construct the
-platform. The ``cl_platform_id`` is retained and available via
-get_. When passed a :ref:`device_selector`, a platform is constructed
-that includes the preferred device.
+The default constructor create a SYCL platform instance that is a
+copy of the platform which contains the device returned by
+``default_selector_v``. When passed a :ref:`device selector<device-selectors>`,
+constructs a SYCL platform instance that is a copy of the platform which
+contains the device returned by the device selector parameter.
 
+===============
+Member function
+===============
 
-``get``
-=======
+``get_backend``
+===============
 
 ::
 
-  cl_platform_id get() const;
+  backend get_backend() const noexcept;
+
+Returns a backend identifying the SYCL backend associated with
+this platform.
+
+``get_info``
+============
 
-Returns the OpenCL device associated with the platform.
+::
 
-Only call this when the platform constructor was passed a
-``cl_platform_id``.
+  template <typename Param>
+  typename Param::return_type get_info() const;
 
-.. _platform-get_devices:
+Queries this SYCL platform for information requested by the template
+parameter ``Param``. The type alias ``Param::return_type`` must be defined in
+accordance with the info parameters in :ref:`device descriptors<device>`
+to facilitate returning the type associated with the ``Param`` parameter.
 
-``get_devices``
-===============
+``get_backend_info``
+====================
 
 ::
 
-  sycl::vector_class<sycl::device> get_devices(
-     sycl::info::device_type = sycl::info::device_type::all) const;
+  template <typename Param>
+  typename Param::return_type get_backend_info() const;
+
 
-Returns vector of SYCL devices associated with the platform and
-filtered by :ref:`info-device_type`
+Queries this SYCL platform for SYCL backend-specific information requested
+by the template parameter ``Param``. The type alias ``Param::return_type`` must be
+defined in accordance with the SYCL backend specification. Must throw an
+exception with the ``errc::backend_mismatch`` error code if the SYCL backend
+that corresponds with ``Param`` is different from the SYCL backend that is
+associated with this platform.
 
 .. rubric:: Example
```

Comment:
This link to the example is confusing because the example does not use `get_backend_info`.  Did you intend to add the link to the section on `get_info` instead?

Reply:
Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/69/commits/1dfdf2e32a01badcbd9c66be95c200dc1768d4a4
The link added in all reflected sections : 
_get_platforms
get_info
get_devices_